src/scripts/main.js

```diff
@@ -1 +1,58 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+function getSuccessMessage(message) {
+  return body.insertAdjacentHTML(
+    'beforeend',
+    `<div data-q="notification" class="message">${message}</div>`
+  );
+}
+
+function getErrorMessage(message) {
+  return body.insertAdjacentHTML(
+    'beforeend',
+    `<div data-q="notification" class="
+     message message--warning">${message.message}</div>`
+  );
+}
```

Comment:
These two function are alike. Try to move it to a single function, than inside this function you can decide if to add 'message-warning' class.

Function's usage might look like this.
`getMessage('Some message', 'success');`
Hint: you may use 2nd argument that will be type of message.

Reply:
Hi, I've changed the code now - let me know. Thanks!